addons/userconfig/cba_settings.sqf

```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
```

Comment:
Unrelated change

Reply:
No need to force this settings server-side, players should be able to use goggle effects if they want to

Reply:
Not related to ACE medical changes